t/unit/backends/test_elasticsearch.py

```diff
@@ -80,3 +80,32 @@ def test_backend_params_by_url(self):
             assert x.scheme == 'elasticsearch'
             assert x.host == 'localhost'
             assert x.port == 9200
+
+    def test_index(self):
+        x = ElasticsearchBackend(app=self.app)
+        x.doc_type = 'test-doc-type'
+        x._server = Mock()
+        x._server.index = Mock()
+        expected_result = dict(
+            _id=sentinel.task_id,
+            _source={'result': sentinel.result}
+        )
+        x._server.index.return_value = expected_result
+
+        body = {"field1": "value1"}
```

Comment:
The test is failing because the `ElasticsearchBackend._index` method has not been called.

```
result = x._index(...)
assert result == expected_result
```